### PR TITLE
Use Python Script Instead of Bash Scripts

### PR DESCRIPTION
**Pull request by ryan-clancy**
@lintool Here's a first stab at the Python script implementation...

Instead of running the bash scripts, you just make sure the `config.json` file points to the index and config file locations for your collection, run the script, and reload the cores from the admin UI.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "solr",
+  "index_mount": "/",
+  "config_mount": "/opt/solr/server/solr/configsets",
+  "indexes": [
+    {
+      "name": "core17",
+      "index_path": "/home/ryan/waterloo/projects/index/lucene-index.core17.pos+docvectors+rawdocs",
+      "config_path": "/home/ryan/waterloo/projects/SolrAnserini/configsets/core17"
```
Comment (lintool) [inline config.json]: Change this to a relative path, relative to current directory, e.g., `configsets/core17`

File: config.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "solr",
+  "index_mount": "/",
+  "config_mount": "/opt/solr/server/solr/configsets",
+  "indexes": [
+    {
+      "name": "core17",
+      "index_path": "/home/ryan/waterloo/projects/index/lucene-index.core17.pos+docvectors+rawdocs",
```
Comment (lintool) [inline config.json]: Change this to a relative path? Defaults shouldn't have tour home directory hard coded?

File: run.py [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    volumes = get_volumes(config)
+    return client.containers.run(config["name"],
+                                 detach=True,
+                                 name=config["name"],
+                                 ports={"8983": "8983"},
+                                 user="solr",
+                                 volumes=volumes)
+
+
+if __name__ == '__main__':
```
Comment (lintool) [inline run.py:101]: Shouldn't the config file be passed in as a config parameter? E.g., `python run.py -config config.json`?

File: config.json (hunk 1 above)
Comment (ryan-clancy) [reply config.json]: Added in latest commits.

File: config.json (hunk 2 above)
Comment (ryan-clancy) [reply config.json]: Added in latest commits.

File: run.py (hunk 3 above)
Comment (ryan-clancy) [reply run.py:101]: Added in latest commits.

File: run.py [hunk 4]
```diff
@@ -0,0 +1,107 @@
+import argparse
+import json
+import os
+import time
+
+import docker
```
Comment (paopao74cn) [inline run.py:6]: This requires installing Docker SDK for Python:
```
pip install docker
```

**Review by paopao74cn — APPROVED** [review]
Just tested. It is working fine! 👍

File: run.py (hunk 4 above)
Comment (ryan-clancy) [reply run.py:6]: Thanks for the catch! Added that to the README

**Review by paopao74cn — APPROVED** [review]
I think it is good to go.

**Pull request merged**